packages/nitro-protocol/gas-benchmarks/fixtures.ts

```diff
@@ -230,3 +243,50 @@ export async function challengeChannelAndExpectGas(
   const finalizesAt = await getFinalizesAtFromTransactionHash(challengeTx.hash);
   return {proof, finalizesAt};
 }
+
+interface ETHBalances {
+  Alice: BigNumberish;
+  Bob: BigNumberish;
+  Ingrid: BigNumberish;
+}
+
+interface ETHHoldings {
+  LforG: BigNumberish;
+  G: BigNumberish;
+  J: BigNumberish;
+  X: BigNumberish;
+}
+
+/**
+ * Asserts the ETH balance of the supplied ethereum account addresses and the ETH holdings in the statechannels asset holding contract for the supplied channelIds.
+ */
+export async function assertETHSanityChecks(
```

Comment:
Given that this is meant to be an "actually specifying test helper", and not just a sanity check, a name like `assertEthBalances` would probably be more appropriate?